### DISCUSSION TITLE
Allow Pronouns to be made public on profile

### DISCUSSION THREAD

**Discussion by jpomfret** [community/community#102493] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hey folks,
I was looking at my GitHub profile while logged out of my account and noticed that my pronouns don't show up - I have them set and they do show up when I'm logged in, but it would be great if there was the opportunity to make these public so they show when anyone views my profile.

![image](https://github.com/community/community/assets/981370/643f0775-4be9-4135-a841-0585020817f5)

I found an old post that labelled this as a bug, but then it was closed as it was 'expected behaviour' - I'm wondering if this could be reinvestigated to allow pronouns to be public?

https://github.com/orgs/community/discussions/78811

Thanks
Jess

**Comment by Windos** [comment]
Onboard 100% with having the _option_ to make pronouns publicly viewable.

The argument that this sort of info is hidden behind authentication (even if the user wants it to be public) so that it isn't scraped, as was mentioned in that issue, doesn't hold water for me considering other scrapable things like social links are visible to unauthenticated visitors.

**Reply by jenkuntz** [reply]
I agree 100%. I already know my pronouns, so only showing them to me or others when I'm logged in seems pointless.
